### ISSUE TITLE
phpmyadmin always starts having issues with my opencart 3.7 after some time; "users" don't open because of errors

### ISSUE THREAD

**Issue by artemraab** [phpmyadmin/phpmyadmin#17053]
**Phpmyadmin info:**
_Database server
    Server: 127.0.0.1 via TCP/IP
    Server type: MariaDB
    Server connection: SSL is not being used Documentation
    Server version: 10.4.20-MariaDB - mariadb.org binary distribution
    Protocol version: 10
    User: root@localhost
    Server charset: UTF-8 Unicode (utf8mb4)
Web server
    Apache/2.4.48 (Win64) OpenSSL/1.1.1k PHP/8.0.8
    Database client version: libmysql - mysqlnd 8.0.8
    PHP extension: mysqli Documentation curl Documentation mbstring Documentation
    PHP version: 8.0.8
phpMyAdmin
    Version information: 5.1.1 (up to date)_

### DESCRIPTION OF THE PROBLEM:
The issue is in accessing User Accounts after using opencart for sometime, I don't know what provokes the error but currently I went there because Facebook ads extension said to check the permission is granted to user account in the database and I can't access the user options to grant the permission to oc_facebook_product as it says.
![image](https://user-images.githubusercontent.com/69615476/128416071-c7d2a22b-a980-4754-ae43-e1c5bcbfb70a.png)
![image](https://user-images.githubusercontent.com/69615476/128416180-838ea186-0793-4619-bd81-bae6f5c1c43c.png)
![image](https://user-images.githubusercontent.com/69615476/128416261-c173bf13-34cc-4c37-9799-737105639cad.png)
It keeps saying:
_#1034 - Index for table 'db' is corrupt; try to repair it_
when I try accessing the opencart3_admin I use as admin for my opencart database.

**Full SQL query is:**
```sql
(SELECT DISTINCT `User`, `Host` FROM `mysql`.`user` ) UNION (SELECT DISTINCT `User`, `Host` FROM `mysql`.`db` ) UNION (SELECT DISTINCT `User`, `Host` FROM `mysql`.`tables_priv` ) UNION (SELECT DISTINCT `User`, `Host` FROM `mysql`.`columns_priv` ) UNION (SELECT DISTINCT `User`, `Host` FROM `mysql`.`procs_priv` ) ORDER BY `User` ASC, `Host` ASC
```

So, what is that 1034 entry that is damaged, how do I fix that damaged line, I can't access User list so I can't edit or delete or create new user for the database... Please help me fix that.

**Comment by williamdes** [comment]
Hi @artemraab 
As you probably already know this is not a phpMyAdmin issue because the error is reported by MySQL but we will still help you with this issue ;)
Did you upgrade your MySQL version ?

I would recommend you to run `mysql_upgrade` on the Database server, it may fix this

**Comment by williamdes** [comment]
This also could help you: https://severalnines.com/database-blog/my-mysql-database-corrupted-what-do-i-do-now

Using `mysqlcheck` tool

**Comment by artemraab** [comment]
> 
> 
> Hi @artemraab
> As you probably already know this is not a phpMyAdmin issue because the error is reported by MySQL but we will still help you with this issue ;)
> Did you upgrade your MySQL version ?
> 
> I would recommend you to run `mysql_upgrade` on the Database server, it may fix this

Hi, I downloaded new xampp of older version (7.3) instead of 8.0 I had and now it somehow overwrote the permissions or something with the users that I granted to opencart3 db and somebody explained to me it was cache conflict and it seems like the installation to new c:/xampp2 folder fixed the c:/xampp folder maria db settings with permissions so the user list now shows fine and i can see and edit priviliges/permissions to my opencart3 db. But my main issue is not fixed yet!!! The facebook ads extension was uninistalled in modifications, modifications were refreshed, the extension was disabled and enabled as well, everything was done two times, page was refreshed but it keeps saying that the permission to CREATE table is not granted to my opencart3_admin that I use for the db opencart3 however, it's granted all permission for this database. Also facebook event manager says that opencart version supported by it are ending with 3.0.2.x and my opencart version is 3.0.3.7 which maybe the issue or not? it's simply CREATE table that doesn't happen... it says JSON.parse error happens when I go to facebook business extension. Why the extension doesn't create the table in my opencart3 db after i follow the pictured instruction on facebook business? any idea how to fix issue with creating oc_facebook_proudct table? **Screenshots:**
![image](https://user-images.githubusercontent.com/69615476/128427349-2ac7282b-11e1-4ae5-ab54-d91786c703f6.png)
![image](https://user-images.githubusercontent.com/69615476/128427374-8ea4ef2d-88c1-4961-a1c6-7efd001e094d.png)
![image](https://user-images.githubusercontent.com/69615476/128427413-cf906c48-0277-403b-907c-834b14be6358.png)
![image](https://user-images.githubusercontent.com/69615476/128427506-abb77946-108d-4a59-84d8-cf9d387542aa.png)
Thanks.

**Comment by williamdes** [comment]
Oh, I know..
This is an issue fixed in next version of phpMyAdmin (see: #16994)
you need to grant on `opencart3\_admin`

Could you replace your phpMyAdmin with the latest 5.1 version in development [(phpMyAdmin 5.1+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.1+snapshot) ?
It should be easier for you to start back the grant process, it will escape properly the `_`

**Comment by artemraab** [comment]
thanks facebook ads issue was fixed successfully. it is explained greatly here: https://github.com/facebookincubator/Facebook-For-OpenCart/issues/5
